sockeye/train.py

```diff
@@ -179,9 +187,14 @@ def main():
                                                lr_scheduler=lr_scheduler,
                                                rnn_forget_bias=args.rnn_forget_bias)
 
-        if args.params:
+        # We may consider loading the params in TrainingModule, for consistency
```

Comment:
+1
train.py is getting quite big over time and this is inherently logic that belongs to the training module.
TrainingModel.module is built in the constructor of TrainingModel but only initialized within fit()/_fit(), so we could check for resume_training at that point.